From 847b71b4e00a9004673cfeb3f995171e32562c43 Mon Sep 17 00:00:00 2001
From: Lin Jianhua <linjh@rock-chips.com>
Date: Fri, 14 Aug 2020 09:39:09 +0800
Subject: [PATCH] ARM: dts: rockchip: rv1109-evb-ddr3-v12-facial-gate: modify
 isp cma buffer size from 256M to 92M

rv1109 facial gate use dual camera that resolution is 1080P,
so only need 92M buffer.

Signed-off-by: Lin Jianhua <linjh@rock-chips.com>
Change-Id: I9a53468b1bf544dbc253ca19907d4ebbdde67e98
---
 arch/arm/boot/dts/rv1109-evb-ddr3-v12-facial-gate.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/rv1109-evb-ddr3-v12-facial-gate.dts b/arch/arm/boot/dts/rv1109-evb-ddr3-v12-facial-gate.dts
index d87b2c840ebf..0bf7e5f8bcf1 100644
--- a/arch/arm/boot/dts/rv1109-evb-ddr3-v12-facial-gate.dts
+++ b/arch/arm/boot/dts/rv1109-evb-ddr3-v12-facial-gate.dts
@@ -120,6 +120,11 @@
 	};
 };
 
+/* isp cma buffer don't fiddle with it, dual camera(1920 * 1080) need 92M buffer */
+&isp_reserved {
+	size = <0x5c00000>;
+};
+
 &mipi_csi2 {
 	status = "okay";
 	ports {
-- 
2.35.3

